cylc/flow/parsec/jinja2support.py

```diff
@@ -207,25 +207,25 @@ def get_error_lines(
     ret = {}
     for line in reversed(traceback.format_exc().splitlines()):
         match = TRACEBACK_LINENO.match(line)
-        lines = None
+        lines: t.List[str] = []
         if match:
             filename = match.groupdict()['file']
             lineno = int(match.groupdict()['line'])
             start_line = max(lineno - CONTEXT_LINES, 0)
-            if filename == '<template>':
+            if filename in {'<template>', '<unknown>'}:
                 filename = base_template_file
                 lineno += 1  # shebang line ignored by jinja2
                 lines = template_lines[start_line:lineno]
-            else:
-                with suppress(FileNotFoundError):  # noqa: SIM117
-                    with open(filename, 'r') as jinja2_file:
-                        jinja2_file.seek(start_line, 0)
```

Comment:
`.seek(n)` goes to byte n, not line n
https://docs.python.org/3/library/io.html#io.IOBase.seek